netx/quicdialer/system_test.go

```diff
@@ -11,6 +11,44 @@ import (
 	"github.com/ooni/probe-engine/netx/trace"
 )
 
+func TestSystemDialerSuccess(t *testing.T) {
```

Comment:
This test seems redundant? Isn't this a subset of `TestSystemDialerSuccessWithReadWrite`?